project/BinaryIncompatibilities.scala

```diff
@@ -26,6 +26,12 @@ object BinaryIncompatibilities {
   )
 
   val Library = Seq(
+    ProblemFilters.exclude[MissingMethodProblem](
+        "scala.scalajs.runtime.EnvironmentInfo.javaSystemProperties"),
+    ProblemFilters.exclude[MissingMethodProblem](
+        "scala.scalajs.runtime.LinkingInfo.linkerVersion"),
+    ProblemFilters.exclude[MissingMethodProblem](
+        "scala.scalajs.runtime.LinkingInfo.scala$scalajs$runtime$LinkingInfo$_setter_$linkerVersion_=")
```

Comment:
Those require a comment explaining why it's fine to exclude them. Here it would be:

    // In theory, breaking, but this is an interface in runtime that no one should extend

like we did in https://github.com/scala-js/scala-js/blob/v0.6.6/project/BinaryIncompatibilities.scala#L121